packages/headless-styles/sandbox/plugins/css-to-ts/convert-inputs/write-file.js

```diff
@@ -18,40 +19,87 @@ const writeToFile = (outPath, input, outputType) => {
     if (path.extname(fileOutPath)) {
       fileOutPath = fileOutPath.slice(0, -3)
     } else {
-      fileOutPath = path.join(fileOutPath, input.filename)
+      fileOutPath = path.join(fileOutPath, convertedCss.filename)
     }
 
     if (!existsSync(fileOutPath)) {
       mkdirSync(fileOutPath)
     }
 
-    Object.entries(input.contents).forEach((key) => {
+    Object.entries(convertedCss.contents).forEach((key) => {
       const body = indentObject(key[1], 2)
+
       writeFileSync(path.join(fileOutPath, `${key[0]}.ts`), 'export default ')
       appendFileSync(path.join(fileOutPath, `${key[0]}.ts`), body)
     })
   } else {
     // Create file for generated simple file
     if (!path.extname(fileOutPath)) {
-      fileOutPath = path.join(fileOutPath, `${input.filename}.ts`)
+      fileOutPath = path.join(fileOutPath, `${convertedCss.filename}.ts`)
     }
 
     if (outputType === 'shakeFile') {
       // Case: "ShakeFile": Export input object key, values as exports to file
       writeFileSync(fileOutPath, '')
-      Object.entries(input.contents).forEach((key) => {
+      Object.entries(convertedCss.contents).forEach((key) => {
         appendFileSync(
           fileOutPath,
-          `export const ${key[0]} =  ${JSON.stringify(key[1])};\n`
+          `export const ${key[0]} = ${JSON.stringify(key[1])};\n`
         )
       })
     } else {
       // Case: "File": Export input object to file
-      const body = indentObject(input.contents, 2)
-      writeFileSync(fileOutPath, generatedDisclaimer)
-      appendFileSync(fileOutPath, 'export default ')
-      appendFileSync(fileOutPath, body)
+      const body = indentObject(convertedCss.contents, 2)
+      const { imports, output } = getImports(body)
+
+      const prettierOptions = prettier.resolveConfig.sync(fileOutPath)
+
+      const fileContents =
+        generatedDisclaimer + imports + '\nexport default' + output
+
+      writeFileSync(fileOutPath, prettier.format(fileContents, prettierOptions))
+      console.groupEnd()
+    }
+  }
+}
+
+function getImports(str) {
+  const imports = {}
+  let output = ''
+
+  for (const line of str.split('\n')) {
+    const match = line.match(/"?composes"?: "(.+) from '(.+)'"/)
+
+    if (match) {
+      const [, name, file] = match
+      const moduleReg = /(..\/)+(.+)\/(.+)\.module\.css/
```

Comment:
My suggestion is assuming the first two dots are intended as the dot character, and not any character in order to match "../(path)/(file)"

This will break if we reference any files in a different path (more likely to be `./` than anything else), in which case, we can make one of the dots optional. I don't think we'll need flexibility beyond that

```suggestion
      const moduleReg = /(\.?\.\/)+(.+)\/(.+)\.module\.css/
```